pkg/services/sqlstore/migrations/apikey_mig.go

```diff
@@ -78,4 +78,12 @@ func addApiKeyMigrations(mg *Migrator) {
 		{Name: "key", Type: DB_Varchar, Length: 190, Nullable: false},
 		{Name: "role", Type: DB_NVarchar, Length: 255, Nullable: false},
 	}))
+
+	mg.AddMigration("Add expires to api_key table", NewAddColumnMigration(apiKeyV2, &Column{
+		Name: "expires", Type: DB_DateTime, Nullable: true,
```

Comment:
I strongly suggest to use a column type of integer instead and store dates as epochs. Reason we've had a lot of problems historically with differences between sqlite, postgres and mysql and our database ORM getting weird results when using a query to check if something has expired by doing a date/time comparison. 

So therefore it's suggested to use an epoch since it's naturally UTC time and removes a lot of headaches. Some examples below.

**user auth tokens:**
 - Migration https://github.com/grafana/grafana/blob/219d711597fb34e53aba065190311af02c887200/pkg/services/sqlstore/migrations/user_auth_token_mig.go#L18-L21
 - Filter by comparing epochs https://github.com/grafana/grafana/blob/219d711597fb34e53aba065190311af02c887200/pkg/services/auth/auth_token.go#L46-L48
https://github.com/grafana/grafana/blob/219d711597fb34e53aba065190311af02c887200/pkg/services/auth/auth_token.go#L110-L114

**login attempts:**
 - Migration https://github.com/grafana/grafana/blob/219d711597fb34e53aba065190311af02c887200/pkg/services/sqlstore/migrations/login_attempt_mig.go#L30
 - FIlter by comparing epochs https://github.com/grafana/grafana/blob/219d711597fb34e53aba065190311af02c887200/pkg/services/sqlstore/login_attempt.go#L73